runelite-client/src/main/java/net/runelite/client/plugins/attackstyles/WarnedStyleHandleType.java

```diff
@@ -0,0 +1,8 @@
+package net.runelite.client.plugins.attackstyles;
+
+public enum WarnedStyleHandleType
+{
+	WARN,
```

Comment:
And how will you disable this? I would also add NONE here, and probably just use same enum for both auto-retalliate and warned styles

Reply:
You can disable everything by just disabling the style. I can't find a reason to have defence enabled, but then disable to do anything with that?

Reply:
Well, before this change you could just untick this, now its not possible anymore

Reply:
Untick what? I changed the 'remove warned styles' to 'how do you want to handle the styles'. You couldn't have defence enabled, but then disable the warning in another setting